client/src/components/Search/SearchBox.js

```diff
@@ -90,6 +91,10 @@ class SearchBox extends Component {
     return this.state.tagWidth + existingPadding;
   }
 
+  calculateInputRightPadding() {
+    return this.state.width < 576 ? 121 : 264;
```

Comment:
Where do `121` and `264` come from? These seem like arbitrary numbers. Maybe add a comment indicating what this is actually doing?

Reply:
Alignment of the 'x' that shows in chrome and matches the location shows in 4.11

![image](https://user-images.githubusercontent.com/4809037/206939282-0a45ce4d-135b-4b55-8555-9798ae900f57.png)

![image](https://user-images.githubusercontent.com/4809037/206939606-7e244f48-3453-44fc-88d6-81325684eeba.png)

Have added comment